### PR TITLE
Refactor deprecated M1 SwipeToDismiss, use M3 SwipeToDismissBox

### LINKED ISSUE #1479: Replace Swipeable with AnchoredDraggable
### Pre-Flight checklist

- [X] Did you check to see if this issue already exists?
- [X] This is a single feature request. (Do not put multiple feature requests in one issue)
- [X] This is not a question or discussion. (Use https://lemmy.ml/c/jerboa for that)

### Describe The Feature Request Below

This is the last part of the deprecated code, since upgrading to the latest Compose

It is also on the blockers for removing material  #1433

https://developer.android.com/develop/ui/compose/touch-input/pointer-input/migrate-swipeable

### PR DESCRIPTION
**Pull request by MV-GH**
Use M3 replacement.

Whats strange is that they named it "Swipe left" but dragging to right is swiping right? So I have corrected the naming to fit the behaviour.

This was the last accompanist dependency. Also the last M1 component.

Fixes #1479

https://github.com/LemmyNet/jerboa/assets/67873169/0a1e92d2-a1ff-4d0c-b9d7-01a4e519598d

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -132,12 +132,6 @@ dependencies {
     implementation("io.noties.markwon:image-coil:4.6.2")
     implementation("io.noties.markwon:linkify:4.6.2")
 
-    // Accompanist
-    implementation("com.google.accompanist:accompanist-pager:0.34.0")
```
Comment (dessalines) [inline app/build.gradle.kts:136]: 🥳

File: app/build.gradle.kts [hunk 2]
```diff
@@ -184,7 +178,7 @@ dependencies {
     implementation("androidx.compose.material:material-icons-extended")
 
     implementation("org.ocpsoft.prettytime:prettytime:5.0.8.Final")
-    implementation("androidx.navigation:navigation-compose")
+    implementation("androidx.navigation:navigation-compose:2.7.7")
```
Comment (dessalines) [inline app/build.gradle.kts:181]: This version should come automatically with bom, no?

Comment (MV-GH) [reply app/build.gradle.kts:181]: It did not for me, must ve come from accompanist

Comment (dessalines) [reply app/build.gradle.kts:181]: Gotcha.

**Pull request merged**